pkg/view/tui/commands/website/new.go

```diff
@@ -0,0 +1,577 @@
+	if args.WebsiteName != "" {
+		// Normalize the provided name
+		normalizedName := strings.TrimPrefix(args.WebsiteName, "./")
+
+		// Check if it's a duplicate
+		for _, name := range existingNames {
+			existingName := strings.TrimPrefix(name, "./")
+			if existingName == normalizedName {
+				return Model{}, fmt.Errorf("website name '%s' already exists", normalizedName)
+			}
+		}
+
+		// Validate the format
+		if !WebsiteNameRegex.MatchString(normalizedName) {
+			return Model{}, fmt.Errorf("website name can only contain letters, numbers, underscores and hyphens")
+		}
+
+		// Check if name starts with a valid character
+		if !WebsiteNameStartRegex.MatchString(normalizedName) {
+			return Model{}, fmt.Errorf("website name must start with a letter or number")
+		}
+
+		// Check if name ends with a valid character
+		if !WebsiteNameEndRegex.MatchString(normalizedName) {
+			return Model{}, fmt.Errorf("website name cannot end with a hyphen")
+		}
+
+		// Check if directory already exists
+		if _, err := fs.Stat(normalizedName); err == nil {
+			return Model{}, fmt.Errorf("website directory '%s' already exists", normalizedName)
+		} else if !os.IsNotExist(err) {
+			return Model{}, fmt.Errorf("failed to check website directory: %w", err)
+		}
+	}
+
+	nameValidator := validation.ComposeValidators(WebsiteNameValidators(existingNames)...)
+	nameInFlightValidator := validation.ComposeValidators(WebsiteNameInFlightValidators()...)
+
+	namePrompt := textprompt.NewTextPrompt("websiteName", textprompt.TextPromptArgs{
+		Prompt:            "What would you like to name your website?",
+		Tag:               "name",
+		Validator:         nameValidator,
+		Placeholder:       "",
+		InFlightValidator: nameInFlightValidator,
+	})
+
+	// collect existing paths
+	existingPaths := []string{}
+
+	for _, website := range config.Websites {
+		if website.Path == "" {
+			existingPaths = append(existingPaths, "/")
+			continue
+		}
+
+		existingPaths = append(existingPaths, website.Path)
+	}
+
+	pathValidator := validation.ComposeValidators(WebsiteURLPathValidators(existingPaths)...)
+	pathInFlightValidator := validation.ComposeValidators(WebsiteURLPathInFlightValidators(existingPaths)...)
+
+	pathPrompt := textprompt.NewTextPrompt("path", textprompt.TextPromptArgs{
+		Prompt:            "What path would you like to use?",
+		Tag:               "path",
+		Validator:         pathValidator,
+		Placeholder:       "",
+		InFlightValidator: pathInFlightValidator,
+	})
+
+	step := StepName
+
+	namePrompt.Focus()
+
+	if args.WebsiteName != "" {
+		namePrompt.SetValue(args.WebsiteName)
+		namePrompt.Blur()
+
+		if len(existingPaths) > 0 {
+			step = StepPath
+
+			pathPrompt.Focus()
+		} else {
+			step = StepTool
+		}
+	}
+
+	if args.WebsitePath != "" {
+		// check if the path is already in use
+		if lo.Contains(existingPaths, args.WebsitePath) {
+			return Model{}, fmt.Errorf("path %s is already in use", args.WebsitePath)
+		}
```

Comment:
```suggestion
		pathInUse := lo.Contains(existingPaths, args.WebsitePath)
		if pathInUse {
			return Model{}, fmt.Errorf("path %s is already in use", args.WebsitePath)
		}
```

I think code should be used over comments to explain what's going on, it may not even be needed here, the `if` statement was already pretty clear.

Please replace/remove the code comments through, anywhere they explain *what* the code is doing.